scripts/run-on-vm.sh

```
@@ -386,7 +391,8 @@ if [[ "$RUN_PHASE" == "all" || "$RUN_PHASE" == "deploy" ]]; then
 
     step "Deploying ptp-operator manifests"
     cd "${PTP_TOOLS_DIR}"
-    run_quiet_with_log_dump_on_failure "make-deploy-all" sh -c "make deploy-all || true"
+    run_quiet_with_log_dump_on_failure "make-deploy-all" \
+      sh -c 'for i in 1 2 3; do make deploy-all && break || { echo "deploy-all attempt $i failed, retrying in 10s..."; sleep 10; }; done'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)